### ISSUE TITLE
Minor things

### ISSUE THREAD

**Issue by davelab6** [enathu/kavivanar#4]
- [x] Font copyright notice does not match OFL.txt notice, which is better and is

```
Copyright (c) 2015, Tharique Azeez (http://thariqueazeez.com|zeezat@gmail.com)
```
- [x] vertical metrics are not right. 

```
$ fontbakery-fix-vertical-metrics.py Kavivanar-Regular.ttf 
WARNING: Kavivanar-Regular.ttf has NOT even heights
Parameter              Kavivanar-Regular.ttf    
ymax                   1065                     
hhea.ascent            952                      
OS/2.sTypoAscender     693                      
OS/2.usWinAscent       952                      
ymin                   -417                     
hhea.descent           -339                     
OS/2.sTypoDescender    -162                     
OS/2.usWinDescent      339                      
hhea.lineGap           0                        
OS/2.sTypoLineGap      59                       
hhea total             1291                     
typo total             914                      
win total              1291                     
UPM:Heights            1000:1482                
UPM:Heights %          148 %                    

High Glyphs
Kavivanar-Regular.ttf: Yacute Gcircumflex Scaron Sacute Cacute Icircumflex Uhungarumlaut Aring Rcaron Agrave Abreve Lacute Gbreve Uring lacute Ncaron uni0206 Ecaron Obreve Ubreve uni01C4 Jcircumflex Ucircumflex Uacute Ocircumflex uni020E uni020C uni020A Zacute Eacute uni0208 uni0204 uni0202 uni0200 Ccaron Acircumflex Tcaron Racute Egrave Ecircumflex Aacute Oacute Dcaron Ugrave Zcaron uni01F4 Igrave Scircumflex Ibreve Ccircumflex Ograve Iacute Nacute Ohungarumlaut

Low Glyphs
Kavivanar-Regular.ttf: tm_CUu tm_TU tm_PUu tm_LllUu tm_NgUu tm_NyU tm_NU tm_Varudam tm_NnnU tm_NnU tm_VUu tm_LU tm_Llla tm_LllU tm_Shree tm_Aa tm_YUu
```

They should be 1065/-417/0
- [x] fsSelection should indicate it is a regular
- [x] italicAngle is -5.0 should be 0

**Comment by davelab6** [comment]
Ah this project has multiple copyright holders, you and vernon adams. so you should move to the 'new style' with `Copyright 2012 The Kavivanar Project Authors`

https://github.com/googlefonts/gf-docs/blob/master/ProjectChecklist.md#ofltxt

**Comment by davelab6** [comment]
- [x] The full license text was in the font but it should be the subset used by GF
  https://github.com/googlefonts/gf-docs/blob/master/ProjectChecklist.md#license

**Comment by davelab6** [comment]
Name IDs 16 and 17 are not needed for Regular/Italic/Bold/BoldItalic sytles

```
    <namerecord nameID="16" platformID="3" platEncID="1" langID="0x409">
      Kavivanar
    </namerecord>
    <namerecord nameID="17" platformID="3" platEncID="1" langID="0x409">
      Regular
    </namerecord>
```

**Comment by davelab6** [comment]
The head table version was 0.1 but should be 1.XX to match the version in name table.

**Comment by davelab6** [comment]
FFTM table should be removed

**Comment by davelab6** [comment]
Here is a hotfix'd ttf 

[Kavivanar-Regular.ttf.zip](https://github.com/enathu/kavivanar/files/204951/Kavivanar-Regular.ttf.zip)

**Comment by davelab6** [comment]
> Ah this project has multiple copyright holders, you and vernon adams. 

Hmm, actually I'm not sure, isn't this 100% your own work?

**Comment by enathu** [comment]
@davelab6 On multiple copyright holders issue, I think I have mistakenly duplicated the OFL license text to this font from one of my fonts - [Pavanam](https://github.com/enathu/pavanam) which is a font with multiple copyright holders. Now I have fixed the copyright reference issues as this font is an original 100% of my own work. I have pushed the latest version to the repo with all the other fixes too. Cheers.
